### ISSUE TITLE
Fix for Drag&Drop Tutorial

### ISSUE THREAD

**Issue by franckyK** [html5rocks/www.html5rocks.com#1482]
On the "[Native HTML5 Drag and Drop](https://www.html5rocks.com/en/tutorials/dnd/basics/)" page the `<header>` texts in all examples are missing, so you can't see what's happening when dragging & dropping.

![drag-and-drop--html5rocks-a](https://user-images.githubusercontent.com/37254170/37247980-efe50e98-24c5-11e8-86cf-ecb841edee93.png)

Culprit is the `header{display:none}` in the general stylesheet. 
Solution: adding `.column header{display:block}` in the inline styles.

![drag-and-drop--html5rocks-b](https://user-images.githubusercontent.com/37254170/37247985-f68d4c56-24c5-11e8-9c54-be4841aaf5c1.png)

Greetings,
franckyK

**Comment by ebidel** [comment]
Thanks for the report. This is an old one :)

 Interested in submitting a PR? I think we should make the demo work again since a lot of content around the web references this tutorial.

**Comment by franckyK** [comment]
Indeed, the baby tut of 2010 is 7.5y old now, and in the meantime much is changed in specs and browsers. But oops, as a non-professional I've a lack of know-how for contributing substantially to an updated version.   
Maybe that could be a three-stage rocket (1) a simple img dropping example, (2) file manupulation examples, and (3) this more complex arranging example.


BTW, I saw that a DnD example in the recent HTML5.2 Recommendation and in the WhatWG specification (same example) is using an empty `<ol>`, in which nothing can be dropped without some additions; see [testpage](http://clba.nl/testing/DnD-example-W3C-and-WhatWG.htm) .
